packages/indy-vdr/src/pool/didIdentifier.ts

```diff
@@ -0,0 +1 @@
+export const DID_INDY_REGEX = /^did:indy:((?:[a-z][_a-z0-9-]*)(?::[a-z][_a-z0-9-]*)):([1-9A-HJ-NP-Za-km-z]{21,22})$/
```

Comment:
Maybe we can move this to the `utlils/did.ts` file next to the `ABBREVIATED_VERKEY_REGEX`?